api/location-api/.env.example

```
@@ -1,19 +1,6 @@
-WEATHER_API_KEY="example-weather-api-key"
-LOCATION_API_KEY="example-location-api-key"
-SERP_API_KEY="example-serp-api-key"
-UPSTASH_REDIS_REST_URL="https://example-redis-url.com"
-UPSTASH_REDIS_REST_TOKEN="example-redis-token"
-PORT = "9494"
-
-WEATHER_URL = "https://example-weather-url.com"
-
+WEATHER_API_KEY=EJ4Gf9sgcuSrSdc3WssJ7f0DR4noh9rZ
+LOCATION_API_KEY=1db4298e26914cea8d5d9fe35ece71ab
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LOCATION_API_KEY key should go before the WEATHER_API_KEY key

(UnorderedKey)